cumulusci.yml

```diff
@@ -9,10 +9,353 @@ project:
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/post/validate_ein_flow
+
+    # Experience Site
+    deploy_network_branding:
+        group: "Grants Management: Experience Site"
+        description: Deploys the NetworkBranding files for the Funding Program Portal.   Must be deployed after creating the "Funding Program Portal" Community.
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/customer_enablement/network_branding
+
+    deploy_grantseeker_sharing_set:
+        group: "Grants Management: Experience Site"
+        description: Deploys the "Grantseeker" Sharing Set for community users.   Must be deployed after deploying the "Grantseeker Plus Login" Profile.
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/customer_enablement/grantseeker_sharing_set
+
+    deploy_guest_user_profile:
+        group: "Grants Management: Experience Site"
+        description: Deploys the "Funding Program Portal Profile" profile for guest users.   Must be deployed after deploying the Funding Program Portal community.
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/customer_enablement/guest_user_profile
+
+    deploy_navigation_menu:
+        group: "Grants Management: Experience Site"
+        class_path: cumulusci.tasks.salesforce.Deploy
+        description: Deploys the "Funding Program Portal Navigation" Navigation Menu for the "Funding Program Portal" Community/Network.   Must be deployed after creating the "Funding Program Portal" Community.
+        options:
+            path: unpackaged/customer_enablement/navigation_menu
+
+    create_grantseeker_community:
+        group: "Grants Management: Experience Site"
+        description: "Creates a community with the 'Grants' template"
+        class_path: cumulusci.tasks.salesforce.CreateCommunity
+        options:
+            template: Grantee Portal
+            name: Funding Program Portal
+            url_path_prefix: fundingprograms
+            timeout: 60000
+
+    publish_grantseeker_community:
+        group: "Grants Management: Experience Site"
+        description: Publishes the "Funding Program Portal" Community in the target org using the Connect API
+        class_path: cumulusci.tasks.salesforce.PublishCommunity
+        options:
+            name: Funding Program Portal
+
+    add_network_members:
+        group: "Grants Management: Experience Site"
+        description: "Create a NetworkMemberGroup record for the community user profile"
+        class_path: cumulusci.tasks.salesforce.network_member_group.CreateNetworkMemberGroups
+        options:
+            network_name: Funding Program Portal
+            profile_names:
+                - Grantseeker Plus Login
+
+    # Verification checks
+    create_verificationitem_document_type:
```

Comment:
Do we need these tasks ? 
`create_verificationitem_document_type`,
`create_taxstatussearch_document_type`

Reply:
@sella24 , see https://salesforce-internal.slack.com/archives/C02R27B4V4H/p1639769086020700